addons/custom-block-shape/addon.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Custom block shape",
+  "description": "Adjust the padding, corner radius, and notch height of Scratch blocks.",
+  "tags": ["editor", "theme", "beta"],
+  "credits": [
+    {
+      "name": "SheepTester",
+      "link": "https://sheeptester.github.io/hello-world/userscripts/"
+    },
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://scratch.mit.edu/users/lisa_wolfgang"
+    }
+  ],
+  "enabledByDefault": false,
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "info": [
+    {
+      "type": "notice",
+      "text": "Setting the padding size to a smaller value may decrease usability.",
```

Comment:
Can you elaborate what "usability" means on here?

Reply:
I'm actually just going to remove that message and replace it with a warning about block positioning.